app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
```

Comment:
Code Style: When importing multiple classes or modules, use parentheses to wrap them. According to the checklist, you should format it like: `from app.errors import (VaccineError, NotWearingMaskError)`.